pages/doc/integrations_tas_alerts.md

```diff
@@ -46,251 +84,437 @@ This metric can indicate that TAS is out of container space or that there is a l
-
-Percentage of the ephemeral disk used on the VM. Set an alert and investigate if the ephemeral disk usage is too high for a job over an extended period.
+**Troubleshooting**
 
-1. Run bosh `vms --details` to view jobs on affected deployments.
-2. Determine the cause of the data consumption, and, if appropriate, increase disk space or scale out the affected jobs.
+* Investigate what is filling the jobs system partition.
 
-This partition should not typically fill because BOSH deploys jobs to use ephemeral and persistent disks.
+## TAS BOSH VM Ephemeral Disk Used
 
-## TAS BOSH VM Health
+Percentage of the ephemeral disk used on the VM. 
 
-This is the most important BOSH metric to monitor. It indicates if the VM emitting the metric is healthy. Review this metric for all VMs to estimate the overall health of the system.
+Investigate if the ephemeral disk usage is too high for a job over an extended period.
 
-* 1 means the system is healthy.
-* 0 means the system is not healthy.
+**Troubleshooting**
 
-Multiple unhealthy VMs signals problems with the underlying IAAS layer.
+* Determine the cause of the data consumption, and, if appropriate, increase disk space or scale the affected jobs.
 
 ## TAS BOSH VM Memory Used
 
 Percentage of memory used on the VM.
 
+**Troubleshooting**
+
+*  The response depends on the job the metric is associated with. If appropriate, scale affected jobs and monitor for improvement.
+
 ## TAS BOSH VM Persistent Disk Used
 
-Percentage of the persistent disk used on the VM. Set an alert and investigate if the persistent disk usage is too high for a job over an extended period.
-1. Run `bosh vms --details` to view jobs on affected deployments.
-2. Determine cause of the data consumption, and, if appropriate, increase disk space or scale out the affected jobs.
+Percentage of the persistent disk used on the VM.
 
-This partition should not typically fill because BOSH deploys jobs to use ephemeral and persistent disks.
+Investigate if the persistent disk usage is too high for a job over an extended period.
+
+**Troubleshooting**
+
+* Determine the cause of the data consumption, and, if appropriate, increase disk space or scale the affected jobs.
+
+## TAS Certificate Expiration
+
+The number of seconds until expiration of each certificate in the foundation. Negative values indicate that the certificate expired in the past.
+
+By default, this alert will fire if any of the certificates on your foundation will expire within 30 days.
+
+By default, entries appear with the following color-coding:
+
+* Red -- Indicates that the expiration time is under 7 days
+* Orange -- Indicates that the expiration time is under 30 days
+* Green -- Indicates that the expiration time is more than 30 days
+
+**Troubleshooting**
+
+* Update your certificates.
+
+## TAS CLI Command Status
+
+The total number of times the App SLI Smoke Test Suite fails. With the default thresholds, the alert will fire after 2 consecutive failures.
```

Comment:
maybe "fails" -> "has failed" is better here too?

Reply:
Sure, I'll replace it.